### ISSUE TITLE
Hide site-name and topnavOpen from keyboard navigation when mobile menu is open

### ISSUE THREAD

**Issue by FennaKort** [FennaKort/php-and-accessible-menu-learning#6]
https://github.com/FennaKort/php-and-accessible-menu-learning/issues/2#issuecomment-3212876167
header>a#site-name is shown to screen-reader users when mobile menu is open
> This is also causing tabbing order issues for keyboard users when mobile menu is open. Using keyboard navigation, I am able to focus on and select header>a#site-name while the mobile menu is open. I am also able to do the same for the topnavOpen button while the menu is open.
> 
> Additionally, showing the tabbing order in firefox devtools while viewing a simulated mobile device causes the following undesired behaviour:
> 
>     * if tabbing order is turned on while mobile menu is open,
> 
>     * and user closes mobile menu then reopens mobile menu using any control method,
> 
>     * mobile menu may reopen as blank screen
>       I can actually still tab through the menu options, and they are shown in the list to screen-reader users, but they are not visible and there is no visual indication of tabbing through either. I do not know if this would occur while actually using keyboard navigation, as I have not been able to replicate it without turning on "show tabbing order" in devtools, but I want to mitigate the chances of this happening to an actual user if at all possible
> 
> 
> <img alt="Image" width="669" height="294" src="https://private-user-images.githubusercontent.com/181121136/480782273-5eab036e-ca17-49d2-b2fe-88e45bb30834.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU5MDk4NTAsIm5iZiI6MTc1NTkwOTU1MCwicGF0aCI6Ii8xODExMjExMzYvNDgwNzgyMjczLTVlYWIwMzZlLWNhMTctNDlkMi1iMmZlLTg4ZTQ1YmIzMDgzNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgyM1QwMDM5MTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00NjYwYzBlYzJiZjA2MjNkZTk5MGZlZTYyYTFkZWVkNTUzYTA3MzhmZjBjY2U3YTIxZDVmNGNkY2YzM2QwZTZmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LBRr6Apw88JQoux5Gf0ckwZZlHXdvLFEDlU7Dmxl5ls">
